pdc/apps/utils/utils.py

```diff
@@ -59,3 +61,33 @@ def read_permission_for_all():
 
 def get_model_name_from_obj_or_cls(obj_or_cls):
     return ContentType.objects.get_for_model(obj_or_cls).model
+
+
+class RelatedNestedOrderingFilter(OrderingFilter):
+    """
+    Extends OrderingFilter to support ordering by fields in related models
+    using Django orm '__', this change is based on rest_framework 3.2.5
```

Comment:
If it works in rest_framework 3.2.5, why is this class needed? I don't see where the actual ordering is done. Shouldn't `filter_queryset()` method be also implemented?